docs/content/preview/migrate/release-notes.md

```diff
@@ -13,6 +13,24 @@ type: docs
 
 Included here are the release notes for the YugabyteDB Voyager v1 release series. Content will be added as new notable features and changes are available in the patch releases of the YugabyteDB v1 series.
 
+## v1.3 - May 26, 2023
+
```

Comment:
@aishwarya24  change the date of the release to `May 30, 2023`

Reply:
Yes Ill keep this one open and change the date when we are ready to release.

Reply:
Okay, thanks!